department-of-veterans-affairs/caseflow
# CASEFLOW-2223 — Fix Bugs with Substitution on Hearing Docket

Connects [CASEFLOW-2223](https://vajira.max.gov/browse/CASEFLOW-2223)

### Description
This fixes a couple of bugs that surfaced when testing appellant substitution on `hearing` docket 

### Acceptance Criteria
- [ ] handle `EvidenceSubmissionWindowTasks` differently when on `hearing` docket vs `evidence_submission` docket
- [ ] display `Hearing` objects from source appeal when viewing substitution/target appeal

### Testing Plan
1. Open up `substitute_appellant_task_copy_spec.rb` file, and remove the `skip`
2. Uncomment the `binding.pry` on line 21
3. Run through the test on line 82 and be sure to select "Evidence Submission Window" during task selection
4. Verify task tree
5. Verify the target appeal stream shows hearing info from source
